docs/sources/writing-guide/documentation-structure/index.md

```diff
@@ -12,21 +12,52 @@ keywords:
 
 # Documentation structure
 
-The technical writing team makes very intentional decisions about how we organize and structure product documentation. Users can find what they need quickly and easily with well-structured content.
+The Grafana Labs Docs team makes intentional decisions about how to organize and structure product documentation. With well-structured content, users can find what they need quickly and easily.
 
-Information architecture (IA) refers to the way in which content is organized. Before you start contributing to product documentation, it is important to understand the IA of the content.
+Before you begin contributing to documentation, it is important to understand the structure of the content.
 
-Generally, the IA determines how content is:
+According to STC’s [Information Design and Information Architecture: Why Technical Communicators Should Care About These Fields](https://www.stc.org/intercom/2022/05/information-design-and-information-architecture-why-technical-communicators-should-care-about-these-fields/),
+information architecture is "...the practice of organizing, structuring, and labeling information to it’s easy to find, use, and understand..."
+
+Generally, documentation structure determines how content is:
 
 - Titled
 - Grouped
 - Combined (or not combined) with other, related content
 
 The following examples are based on the Grafana OSS and Enterprise documentation.
 
-## Information architecture
+## Structure of published content
+
+Using the same content structure across documentation provides consistent experience. Topics are listed from high level to more specific. For example, a new Grafana user may wish to learn conceptual information first, so Introduction is listed before installation.
```

Comment:
```suggestion
A content structure that spans sets of documentation provides a consistent experience.
Information flows from a higher (less specific) level to a lower (more specific) level.
For example, a new Grafana user wants to learn conceptual information first, so _Introduction_ comes before _Installation_.
```

Reply:
(I also introduced semantic line breaks here.)

Reply:
Can we add some notes about a content framework supporting customers along their entire journey - from pre-sales thru config thru upgrade - to link this to our model?

Reply:
See #210.